app/components/list-item/index.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { render, screen } from '@testing-library/react'
+import ListItem from '.'
+import { IListItemType } from '@/app/app-interfaces'
+import { faker } from '@faker-js/faker'
+import { appLabels } from '@/app/constants'
+
+describe('List-item Component', () => {
+  const todoItems: IListItemType[] = [
+    {
+      id: faker.string.uuid(),
+      name: faker.word.words(),
+    },
+    {
+      id: faker.string.uuid(),
+      name: faker.word.words(),
+    },
```

Comment:
SUGGESTION - you can use faker.helper.multiple() to avoid repetition here while creating array.